src/main/java/in/org/projecteka/hiu/consent/model/ConsentRequestRepresentation.java

```diff
@@ -21,7 +21,7 @@ public static ConsentRequestRepresentation toConsentRequestRepresentation(
             Patient patient,
             in.org.projecteka.hiu.consent.model.ConsentRequest consentRequest) {
         var format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
-        var withMillSeconds = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+        var withMillSeconds = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'+0000'");
```

Comment:
Why we are changing this here?

Reply:
That's how the actual date is getting stored in string format